lib/koans/09_map_sets.ex

```diff
@@ -9,6 +9,10 @@ defmodule MapSets do
     assert Enum.fetch(@set, 0) == {:ok, ___}
   end
 
+  koan "How large is my map set?" do
```

Comment:
I don't like that this koan moved here interrupts the conversation of the others it falls between.

"I am very similar to a list" -> "However, I do not allow duplication"

Regardless, you are definitely onto something as "However, I do not allow duplication" uses MapSet.size/1 before it's introduced. What do you think about just removing the explicit "How large is my map set?" altogether and let that be something to glean from "However, I do not allow duplication"?

Reply:
Sounds good to me!